xml/System.Configuration.Internal/DelegatingConfigHost.xml

```diff
@@ -192,9 +192,9 @@
       </Parameters>
       <Docs>
         <param name="encryptedXml">An encrypted section of a configuration file.</param>
-        <param name="protectionProvider">A <see cref="T:System.Configuration.ProtectedConfigurationProvider" /> object.</param>
-        <param name="protectedConfigSection">A <see cref="T:System.Configuration.ProtectedConfigurationSection" /> object.</param>
-        <param name="protectedSection">To be added.</param>
+        <param name="protectionProvider">An object containing the providers that encrypt and decrypt protected configuration data.</param>
+        <param name="protectedConfigSection">An object that provides programatic access to the <c>configProtectedData</c> configuration section.</param>
```

Comment:
This parameter had a ref vs src mismatch at some point, causing this xml to be generated with both parameter names.

Same problem applies to the next two diffs in this file.

Reply:
So I presume this will be fixed?

Reply:
I don't think these get fixed because I it seems we published them in a particular .NET version, so we still need to keep that old parameter name there for when the user selects that particular .NET version from the dropdown in MS Docs.